rpm/skopeo.spec

```diff
@@ -67,7 +67,6 @@ BuildRequires: go-rpm-macros
 %endif
 BuildRequires: gpgme-devel
 BuildRequires: libassuan-devel
-BuildRequires: pkgconfig(devmapper)
```

Comment:
Cc: @lsm5 

Also @cevich — maybe the dependency can be removed from some of the `automation_images` as well.

Reply:
Thanks for thinking of the CI VMs.  We do install `device-mapper-devel` in them, but the images are used across many repos.  W/o trying or checking every single one of them, I can't think of an easy way to confirm :thinking:  I think I'm more comfortable leaving it there, and up to some future CI VM package-pruning effort.  Though please let me know if you object for any reason.